vmm/src/config.rs

```diff
@@ -1554,6 +1566,22 @@ impl VsockConfig {
     }
 }
 
+impl PvPanicConfig {
+    pub fn parse(device_type: &str) -> Result<Self> {
+        let mut parser = OptionParser::new();
+        parser.add_valueless("pci");
+        parser.parse(device_type).map_err(Error::ParsePvPanic)?;
+
+        if parser.is_set("pci") {
```

Comment:
I don't think we will ever support anything other than a PCI device so you don't need this.